tornado-api/src/main/java/uk/ac/manchester/tornado/api/profiler/ChromeEventJSonWriter.java

```diff
@@ -0,0 +1,80 @@
+package uk.ac.manchester.tornado.api.profiler;
```

Comment:
Add license. This should be the same as any other file from the API module. 
Example: 
https://github.com/beehive-lab/TornadoVM/blob/master/tornado-api/src/main/java/uk/ac/manchester/tornado/api/GridTask.java#L1-L41

Reply:
Will do